src/Microsoft.Azure.WebJobs.StorageProvider.Blobs/BlobServiceClientProvider.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Storage.Blobs;
+using Microsoft.Azure.WebJobs.Shared.StorageProvider;
+using Microsoft.Extensions.Azure;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.StorageProvider.Blobs
+{
+    /// <summary>
+    /// Provider to create BlobServiceClient objects with the registered <see cref="IConfiguration"/>
+    /// </summary>
+    public class BlobServiceClientProvider : StorageClientProvider<BlobServiceClient, BlobClientOptions>
+    {
+        /// <summary>
+        /// BlobStorageClientProvider
+        /// </summary>
+        /// <param name="configuration">Registered <see cref="IConfiguration"/></param>
```

Comment:
nit: better description for the parameters as well. Unclear what `Registered` would mean if actually using this constructor.